### PR TITLE
Add a columnFamily feature for client-go when use rawput in tikv

### PR DESCRIPTION
**Pull request by joccau**
The function of pr: As title
 close https://github.com/tikv/client-go/issues/424

### CODE REVIEW

File: internal/mockstore/mocktikv/mvcc_leveldb.go [hunk 1]
```diff
@@ -61,6 +61,7 @@ import (
 var (
 	LockAlwaysWait = int64(0)
 	LockNoWait     = int64(-1)
+	default_cf     = "test_cf"
```
Comment (kennytm) [inline internal/mockstore/mocktikv/mvcc_leveldb.go:64]:
```suggestion
	defaultCF      = "test_cf"
```

**Review by kennytm — APPROVED** [review]

File: rawkv/rawkv.go [hunk 2]
```diff
@@ -67,8 +67,7 @@ const (
 )
 
 type scanOptions struct {
-	KeyOnly      bool
-	ColumnFamily string
+	KeyOnly bool
```
Comment (disksing) [inline rawkv/rawkv.go:70]: I think the option is still useful. There could be some case that need to read/write different CFs. Creating multiple clients could waste more resource.

File: internal/mockstore/mocktikv/mvcc.go [hunk 3]
```diff
@@ -286,15 +286,15 @@ type MVCCStore interface {
...
-	RawGet(key []byte) []byte
-	RawBatchGet(keys [][]byte) [][]byte
-	RawScan(startKey, endKey []byte, limit int) []Pair        // Scan the range of [startKey, endKey)
-	RawReverseScan(startKey, endKey []byte, limit int) []Pair // Scan the range of [endKey, startKey)
-	RawPut(key, value []byte)
-	RawBatchPut(keys, values [][]byte)
-	RawDelete(key []byte)
-	RawBatchDelete(keys [][]byte)
-	RawDeleteRange(startKey, endKey []byte)
+	RawGet(key []byte, cf string) []byte
```
Comment (disksing) [inline internal/mockstore/mocktikv/mvcc.go:289]: I would like to put CF to the first position.

**Comment by disksing** [comment]
Hi @joccau there some lint warnings. We need them be fixed before merging this PR. Thanks!

**Comment by disksing** [comment]
Thanks @joccau

**Pull request merged**